protocol/messenger_handler.go

```diff
@@ -421,8 +421,9 @@ func (m *Messenger) HandleSyncInstallationPublicChat(state *ReceivedMessageState
 	chat := existingChat
 	if !ok {
 		chat = CreatePublicChat(chatID, state.Timesource)
+		chat.Joined = int64(message.Clock)
 	} else {
-		existingChat.Joined = int64(state.Timesource.GetCurrentTime())
+		existingChat.Joined = int64(message.Clock)
```

Comment:
chat.Joined ?

Reply:
what do you mean?

Reply:
Oh sorry, I should have included more details.

We are modifying `existingChat.Joined` but we are not using it 🤔, should we also change this into `chat.Joined`? then we can just also remove if else check.

Reply:
Oh sorry, I should have included more details.

We are modifying `existingChat.Joined` but we are not using it 🤔, should we change this into `chat.Joined` then, we can also remove if else check

Reply:
both are pointers to the same value, so I doubt there is any difference